canonical/snapcraft.io
# chore: update internal deps

## Done
- Upgraded:
   - vanilla-framework
   - store-components
   - sass
 - fixed sass warning related with snapcraft such as:
```
Deprecation Warning: darken() is deprecated. Suggestions:

color.scale($color, $lightness: -10.3238866397%)
color.adjust($color, $lightness: -10%)

More info: https://sass-lang.com/d/color-functions

   ╷
82 │         background-color: darken($color-light, 10%);
   │                           ^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
    static/sass/_snapcraft_details_heading.scss 82:27  snapcraft-details-heading()
    static/sass/styles-embedded.scss 20:1              root stylesheet
    
```
## How to QA
- Go to https://snapcraft-io-4886.demos.haus/snap-id 
- the install button gets darken when focus
- Go to https://snapcraft-io-4886.demos.haus/snap-id  
-  the releases button focus state should have a little darker color
- click on the releases  button
-  verify the popup has  shadow
- Go to https://snapcraft-io-4886.demos.haus/snap-id/listing
- tour should have a transparent background

## Testing

## Issue / Card
Fixes [#WD-15928](https://warthogs.atlassian.net/browse/WD-15928)
